javascript/ql/src/Declarations/SuspiciousMethodNameDeclaration.ql

```diff
@@ -0,0 +1,69 @@
+/**
+ * @name Suspicious method name declaration
+ * @description Declaring a class or interface method with a special name may cause a normal 
```

Comment:
I still find this to be a messy formulation, but I am out of alternatives. Perhaps the docreview can inspire us.

Reply:
`A method declaration with a name that would be a special keyword in another context is suspicious.` ?
Still feels messy.

Reply:
Yes. Lets go with that.